File: 2023/challenge_01/js/main.js
const URL = 'https://codember.dev/data/message_01.txt';

fetch(URL)
  .then(response => response.text())
  .then(data => {
    const items = data.split(' ');
    const maps = {};

    for (const item of items) {
      // serialization
      const serializedItem = item.toLowerCase().trim();

      if (!(serializedItem in maps)) {
        maps[serializedItem] = 1;
      } else {
        maps[serializedItem] += 1;
      }
    }

    for (const [key, value] of Object.entries(maps)) {
      //console.log(`${key}${value}`);
      process.stdout.write(`${key}${value}`);
    }
  })
  .catch(error => console.error('Error:', error));

File: 2023/challenge_01/python/main.py
import requests

URL = 'https://codember.dev/data/message_01.txt'

req = requests.get(URL)
items = req.text.split(' ')

MAPS = {}

for item in items:
  # serialization
  item = item.lower().strip()

  if item not in MAPS:
    MAPS[item] = 1
  else:
    MAPS[item] = MAPS[item] + 1

for key, value in MAPS.items():
  print(f"{key}{value}", end='')